src/lib/components/Card/cardOptions.ts

```diff
@@ -0,0 +1,3 @@
+export const variants = ["primary", "secondary"] as const;
+
+export type Variant = (typeof variants)[number];
```

Comment:
I initially had this when I assumed there would be more variations dependent on classes, but with the named slots and the only image version of the card being the flag orientation, I think we can remove this entirely for now.